tests/testthat/test-kernel.r

```diff
@@ -0,0 +1,29 @@
+context('kernel')
+
+Sys.setenv(
+    PYTHONWARNINGS = 'ignore::DeprecationWarning',
+    PYTHONPATH = paste(getwd(), Sys.getenv('PYTHONPATH'), sep = ':'))
```

Comment:
This is wrong on all my R environments :-(

I currently code in RStudio and when starting RStudio, no PYTHONPATH is available. The default environment, which is in PATH also doesn't have the kernel testing tool installed, that is in an conda env which I have to activate first.